clap_complete/Cargo.toml

```diff
@@ -40,6 +40,7 @@ is_executable = { version = "1.0.1", optional = true }
 shlex = { version = "1.3.0", optional = true }
 completest = { version = "0.4.2", optional = true }
 completest-pty = { version = "0.5.5", optional = true }
+home = { version = "0.5.0", optional = true }
```

Comment:
With `std::env::home_dir` being undeprecated, we should be using this through a polyfll.  I'll need to go create one before this can move forward.

Reply:
(if it wasn't for that, I'd be hesitant to add this out of question of whether the extra dep is worth it)

Reply:
Perhaps we could copy that function from std? https://github.com/rust-lang/rust/blob/2682b88c526d493edeb2d3f2df358f44db69b73f/library/std/src/sys/unix/os.rs#L595